### DISCUSSION TITLE
Extend the Login Timeout?

### DISCUSSION THREAD

**Discussion by eastedentech** [ngoduykhanh/PowerDNS-Admin#1043] [Q&A] [answered]
Is the login timeout a configurable setting? 
If possible, I would like to extend it to a few hours to minimize the number of times I need to re-authenticate during the day...

-JF

**Comment by ghost** [comment, marked as answer]
Hi @eastedentech,

the session timeout is configurable via the Web UI, if you go to the menu on the left to Settings -> Basic, look for `session_timeout` (the number is in minutes).

**Reply by firstkevinds** [reply]
Warning...  Do not set to 0 (to disable).  Can not login at all afterwards.  Getting 403 errors..  Oops..
